### PR TITLE
[DNM] *: Enable Time for PerfContext

### PR DESCRIPTION
**Pull request by breeswish**
Signed-off-by: Breezewish <breezewish@pingcap.com>

### What problem does this PR solve?

Experiment to enable Time for PerfContext, see whether it's worth to do.

### CODE REVIEW

File: src/storage/kv/perf_context.rs [hunk 1]
```diff
@@ -31,6 +31,7 @@ impl slog::KV for PerfStatisticsInstant {
 impl PerfStatisticsInstant {
     /// Create an instance which stores instant statistics values, retrieved at creation.
     pub fn new() -> Self {
+        set_perf_level(PerfLevel::EnableTimeExceptForMutex);
```
Comment (yiwu-arbug) [inline src/storage/kv/perf_context.rs:34]: Great! How about we make it a config? raftstore has a similar config: https://github.com/tikv/tikv/blob/master/components/raftstore/src/store/config.rs#L184

Comment (breeswish) [reply src/storage/kv/perf_context.rs:34]: @yiwu-arbug Yes! If the performance degrade is notable then it needs a config to opt-in/out.

**Comment by breeswish** [comment]
I think our benchmark infra is broken..

**Pull request closed without merging**